R/vdiagram-data.R

```diff
@@ -1,33 +1,45 @@
+#' Parse Vdiagram Data
+#' @description Takes in a V diagram report and returns the necessary data for a V Diagram
+#' @param reportObject An R object with the raw data required for a V Diagram
+#' @return A list containing the relevant data for a V Diagram
+#'
 parseVDiagramData <- function(reportObject){
   ratingShifts <- fetchRatingShifts(reportObject)
   
-  shiftPoints <- ratingShifts$shiftPoints
+  ### Which of these are required? None of them stop if they're not valid currently.
+  ### Alternatively, do we want to assign validParam(blah, blah) to each (i.e. shiftPoints <- validParam(etc,etc))
+  ### I assume we do as otherwise the "validParam" statements do nothing.
```

Comment:
We should ask Laura to double check this list (using JSON list names). I only did the last three.

```
ratingsShifts >> shiftPoints
ratingsShifts >> stagePoints
ratingsShifts >> shiftNumber
measurements >> errorMaxShiftInFeet 
measurements >> errorMinShiftInFeet
measurements >> shiftInFeet
measurements >> shiftNumber
measurements >> meanGageHeight
measurements >> measurementNumber
measurements >> historic (NOT REQUIRED)
maximumStageHeight (REQUIRED)
minimumStageHeight (REQUIRED)
```